tests/upgrade.bats

```diff
@@ -64,9 +67,8 @@ source $BATS_TEST_DIRNAME/common.bash
 }
 
 @test "[CRD upgrade] Privileged pod should be launched after delete old policies" {
-	kubectl --context $CLUSTER_CONTEXT delete --wait --ignore-not-found -n kubewarden clusteradmissionpolicies --all
-	kubectl --context $CLUSTER_CONTEXT create namespace testns
-
-	kubectl_namespace_apply_should_succeed $RESOURCES_DIR/violate-privileged-pod-policy.yaml testns
+	kubectl delete --wait --ignore-not-found -n kubewarden clusteradmissionpolicies --all
+	kubectl create namespace testns
+	kubectl --namespace testns apply --wait -f $RESOURCES_DIR/violate-privileged-pod-policy.yaml
```

Comment:
how do we know this succeeded?

Reply:
Same as above, bats automatically fails on non 0 exit code if you don't use `run`